Gemfile

```diff
@@ -78,7 +78,7 @@ gem 'dotenv-rails', require: 'dotenv/rails-now'
 
 group :test do
   # Test coverage
-  gem 'coveralls', require: false
+  # gem 'coveralls', require: false
```

Comment:
Please delete this line instead of just commenting it out, and rerun bundle install

(and the comment above as well)